Makefile

```diff
@@ -37,18 +45,6 @@ venv: $(VIRTUALENV_REQUIREMENTS) bin/venv-update
 	    pip-command= pymonkey pip-custom-platform -- pip-faster install --upgrade --prune
 	rm -rf $@/local
 
-.PHONY: install-hooks
-install-hooks: venv
-	./venv/bin/pre-commit install -f --install-hooks
-
-.PHONY: serve
-serve: development
-	dev_appserver.py .
```

Comment:
`dev_appserver.py` is a globally installed python tool from [Google](https://cloud.google.com/appengine/docs/standard/python/tools/using-local-server).  This should not be deleted.